core/camel-api/src/main/java/org/apache/camel/spi/StreamCachingStrategy.java

```diff
@@ -117,38 +117,44 @@ interface SpoolRule {
     boolean isEnabled();
 
     /**
-     * To filter stream caching of a given set of allowed/denied classes. By default, all classes that are
-     * {@link java.io.InputStream} is allowed.
+     * To filter stream caching of a given set of allowed/denied classes. Classes inheriting from any of the specified
+     * classes will be cached.
+     * By default, all classes that inherit from {@link java.io.InputStream} are allowed.
```

Comment:
it is not only rewording, it is adding more information on which class are cached. Given that you mention that it is a "reword", is it expected?